include/tachimawari_dynamixel/dynamixel.hpp

```diff
@@ -123,9 +125,9 @@ class Dynamixel : public rclcpp::Node
     int data_length = 4);
 
   bool move_joint(Joint joint);
-  bool move_joints(std::vector<Joint>);
+  bool move_joints(std::vector<Joint> & joints);
```

Comment:
Actually it's not a good behavior to pass a parameter by reference and changes it inside the function, better to use a `shared_ptr`. But it's okay, it could be solved on another PR.